homeassistant/components/litterrobot/coordinator.py

```diff
@@ -48,6 +48,8 @@ async def _async_update_data(self) -> None:
         """Update all device states from the Litter-Robot API."""
         await self.account.refresh_robots()
         await self.account.load_pets()
+        for pet in self.account.pets:
+            await pet.fetch_weight_history()
```

Comment:
is this still required? if so, maybe add a comment explaining that its needed for `get_visits_since`.

Reply:
Added a comment to explain the necessity here.